vignettes/articles/cookbook_interpretation.Rmd

```diff
@@ -110,37 +119,48 @@ qplot_trajectory(
+The primary variable of interest in the output is `emission_factor_value`, which represents an emission intensity depending on the possible values of the `emission_factor_metric` column`:
 
-If you use the `by_company = TRUE` option of the `target_sda()` function, most of what's said above is true except relative to the company listed in that row in the `name_abcd` column.
+- `"projected"`: The portfolio’s physical emission intensity based on ABCD data for the sector, year, region, and scenario.
+- `"target_*"` (e.g., `"target_cps"`): The scenario-aligned target emission intensity, starting from the portfolio’s initial intensity and converging with the sector-level scenario target by the final year.
+- `"corporate_economy"`: The weighted average emission intensity of the entire sector, serving as a benchmark for comparison.
+- `"adjusted_*"`: The scenario trajectory for emission intensity, adjusted for differences between ABCD sector totals and scenario data.
+
+When `by_company = TRUE`, the output is specific to the company listed in the `name_abcd` column.
 
 For detailed information about the table output of the `target_sda()` function and its columns, see the [data dictionary](data_outputs.html#target_sda-output).
 
 ### Emission Intensity Plot
 
-The emission intensity metric measures the average CO~2~ intensity of the portfolio in the `r cli::format_inline("{unique(co2_intensity_scenario_demo$sector)} sector{?s}")`. This projected emission intensity of a portfolio or company is given as CO~2~/economic unit of output (for example, CO~2~ per ton of steel produced). This five year forward-looking trajectory of the emission intensity is then compared to an emission intensity reference trend over the same period of time based on a climate transition scenario.
+The emission intensity metric measures the average CO~2~ intensity of a portfolio in the `r cli::format_inline("{unique(co2_intensity_scenario_demo$sector)} sector{?s}")`, expressed as CO~2~ per unit of economic output (e.g., CO2 per ton of steel produced). This five-year forward-looking trajectory is compared to an emission intensity reference trend based on a climate transition scenario.
```

Comment:
```suggestion
The emission intensity metric measures the average CO~2~ intensity of a portfolio in the aviation, cement, and steel sectors, expressed as CO~2~ per unit of economic output (e.g., CO2 per ton of steel produced). This five-year forward-looking trajectory is compared to an emission intensity reference trend based on a climate transition scenario.
```
listing the sectors based on the demo data is not a good idea, because the aviation sector is not included in that data set. I would suggest we hard-code the sectors for now,  which is not ideal, but correct